web-components/src/theme/index.css

```diff
@@ -16,3 +17,16 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+/* for some reason this isn't setting appropriately through MUI theme */
+a {
```

Comment:
We manually set these link styles many places in the app, so I decided to pull them into the shared theme CSS instead